### PR TITLE
WL-1872 |  After purchasing a journal, "My Journals" tab should be selected in dashboard 

### PR DESCRIPTION
**Pull request by HammadAhmadWaqas**
After purchasing a journal and clinking on "Go to dashboard" button, 
now "My Journals" tab is selected in dashboard

### CODE REVIEW

File: ecommerce/extensions/checkout/tests/test_views.py [hunk 1]
```diff
@@ -386,6 +391,26 @@ def test_dashboard_link_for_course_purchase(self):
...
+        The dashboard link at the bottom of the receipt for a journal purchase
+        should point to the user journal dashboard."""
+        order = self._create_order_for_receipt(self.user)
+
+        # modify the order's product_class such that it behaves like it has a journal product
+        product_class = order.basket.lines.first().product.get_product_class()
+        product_class.name = JOURNAL_PRODUCT_CLASS_NAME
+        product_class.save()
+
+        response = self._get_receipt_response(order.number)
```
Comment (ziafazal) [inline ecommerce/extensions/checkout/tests/test_views.py:406]: nit:Can we move this line downwards just above the `self.assertEqual(response.status_code, 200)`

**Review by ziafazal — APPROVED** [review]
👍

**Review by bill-filler — APPROVED** [review]

**Comment by HammadAhmadWaqas** [comment]
@georgebabey could you please merge this branch? I do not have permission to merge.

**Comment by georgebabey** [comment]
Codecov report: https://codecov.io/github/edx/ecommerce/commit/dc2c9b64f503b3e55e93d556cd22249331bdb40d

**Pull request merged**